block/local/adapter.go

```diff
@@ -70,9 +70,24 @@ func NewAdapter(path string, opts ...func(a *Adapter)) (*Adapter, error) {
 	}
 	return adapter, nil
 }
+func resolveNamespace(obj block.ObjectPointer) (block.QualifiedKey, error) {
+	qualifiedKey, err := block.ResolveNamespace(obj.StorageNamespace, obj.Identifier)
+	if err != nil {
+		return qualifiedKey, err
+	}
+	if qualifiedKey.StorageType != block.StorageTypeLocal {
+		return qualifiedKey, block.ErrInvalidNamespace
```

Comment:
How can this happen?

Reply:
@arielshaqed 
This could happen if the repository is configured with a different namespace.
Will happen due to wrong configuration (that we currently aloud).
Or, for example, an old repository that was configured when we where using S3, and then we changed to local